Apps/W1/HybridGP/app/src/Migration/Items/GPIV00101.Table.al

```diff
@@ -0,0 +1,377 @@
+table 40116 "GP IV00101"
```

Comment:
Just an FYI. the "GP Item" is the current table used to pull some of this data. Not sure the exact data required, but it might already be available in that table.

Reply:
I wasn't able to use "GP Item" because it didn't have the item class Id (ITMCLSCD) field.